src/fragments/lib/auth/common/mfa/flows.mdx

```diff
@@ -101,9 +113,49 @@ import iosConfirmSignInTOTP from "/src/fragments/lib/auth/ios/sms/confirm_sign_i
 
 <Fragments fragments={{ios: iosConfirmSignInTOTP}} />
 
+import android9 from "/src/fragments/lib/auth/android/sms/confirm_sign_in.mdx";
+
+<Fragments fragments={{android: android9}} />
+
 After a user has been signed in, call `updateMFAPreference` to record the MFA type as enabled for the user
 and optionally set it as preferred so that subsequent logins default to using this MFA type.
 
+<InlineFilter filters={['android']}>
+<BlockSwitcher>
+<Block name="Java">
+
+```java
+if (Amplify.Auth.getPlugin("awsCognitoAuthPlugin") instanceof AWSCognitoAuthPlugin) {
+            AWSCognitoAuthPlugin plugin = (AWSCognitoAuthPlugin) Amplify.Auth.getPlugin("awsCognitoAuthPlugin");
+            plugin.updateMFAPreference(
+                    MFAPreference.ENABLED,
```

Comment:
This will set SMS as preferred MFA. We might want to set TOTP as preference here?